torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6272,6 +6278,18 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                      dtypes=all_types_and_complex_and(torch.bool),
                      default_test_dtypes=floating_and_complex_types(),
                      check_batched_gradgrad=False),
+    SpectralFuncInfo('fft.hfftn',
```

Comment:
For completeness we should add hfft2 and ihfft2, too (this can be done in a follow-up PR)